wiki/Community/User_title/en.md

```diff
@@ -43,6 +43,13 @@ On the forum sections of the [old osu! website](https://old.ppy.sh)<!-- TODO: wi
 
 These titles were only visible on the sidebars of forum posts (not on the users' profiles) and were not shown if another custom title was already set prior.
 
+Post count titles were discontinued with the release of the new osu! website.
+
 ## Trivia
 
 - User titles are referred to as "rank titles" throughout the game's code. This is a leftover name from when osu! was built on top of [phpBB](https://www.phpbb.com/).
+
+- As of March 2021, If a user is eligible to multiple user titles, they would be displayed together on their profile, separated by a slash, instead of only displaying their most recent user title. <!-- VINXIS was the first person to get this treatment when he joined BN, unfortunately the tweet that shows this is deleted -->
```

Comment:
```suggestion
- As of March 2021, if a user is eligible for multiple user titles, they are displayed together on their profile separated by a slash, instead of only displaying their most recent user title.<!-- VINXIS was the first person to get this treatment when he joined BN, unfortunately the tweet that shows this is deleted -->
```